go.mod

```diff
@@ -29,15 +29,15 @@ require (
 
 require (
 	cosmossdk.io/errors v1.0.0-beta.7
-	cosmossdk.io/math v1.0.0-beta.4
+	cosmossdk.io/math v1.0.0-rc.0
 	github.com/99designs/keyring v1.2.1
 	github.com/binance-chain/tss-lib v1.3.2
-	github.com/btcsuite/btcd v0.22.2
+	github.com/btcsuite/btcd v0.23.4
```

Comment:
@brewmaster012 Can you verify this change will not impact in any way

Reply:
actually the version of btcd is pinned down to 
`github.com/btcsuite/btcd => github.com/btcsuite/btcd v0.22.1`
by the replace directive